index.js

```diff
@@ -338,6 +340,14 @@ class HandlebarsRenderer {
             }
         };
     }
+
+    /**
+     * 
+     * @param {String} level 
+     */
+    setLoggerLevel(level) {
+        this.handlebars.logger.level = level;
```

Comment:
Not sure that this will work for `console`. Can you write a test for that?

Reply:
sure, added! 
This behaviour is possible due to this if condition:
https://github.com/bigcommerce/paper-handlebars/blob/master/index.js#L332